scripts/update_version.py

```
@@ -0,0 +1,65 @@
+import argparse
+from os.path import abspath
+
+def read_version(version_file):
+    VERSION_MAJOR = 0
+    VERSION_MINOR = 0
+    VERSION_BUILD = 0
+    VERSION_ALPHA = 0
+
+    with open(version_file, 'r') as file:
+        content = file.read()
+    for l in content.split("\n"):
```

Ruff

Ambiguous variable name: `l`

(E741)